From 4915c8c099b38438673579f26c3de9185f3f39f5 Mon Sep 17 00:00:00 2001
From: Dharageswari R <dharageswari.r@intel.com>
Date: Fri, 15 Dec 2006 12:19:15 +0530
Subject: [PATCH 0232/1240] ASoC: Intel: Modify Icelake machine id to use RT274

Icelake features rt274 codec. Hence updating to use the
right machine driver

Change-Id: Ia60530a67b17b682e9a265150b53e1f2fa7095e2
Signed-off-by: Dharageswari R <dharageswari.r@intel.com>
Reviewed-by: Koul, Vinod <vinod.koul@intel.com>
Reviewed-by: Singh, Guneshwor O <guneshwor.o.singh@intel.com>
Reviewed-by: Kale, Sanyog R <sanyog.r.kale@intel.com>
Reviewed-by: Diwakar, Praveen <praveen.diwakar@intel.com>
Tested-by: Avati, Santosh Kumar <santosh.kumar.avati@intel.com>
---
 sound/soc/intel/skylake/skl.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/sound/soc/intel/skylake/skl.c b/sound/soc/intel/skylake/skl.c
index 07fa286..8e7e362 100644
--- a/sound/soc/intel/skylake/skl.c
+++ b/sound/soc/intel/skylake/skl.c
@@ -1059,7 +1059,7 @@ static const struct sst_acpi_mach sst_cnl_devdata[] = {
 };
 
 static struct sst_acpi_mach sst_icl_devdata[] = {
-	{ "dummy", "icl_wm8281", "intel/dsp_fw_icl.bin", NULL, NULL, NULL },
+	{ "dummy", "icl_rt274", "intel/dsp_fw_icl.bin", NULL, NULL, NULL },
 	{}
 };
 
-- 
2.7.4

